### PR TITLE
Remove some code with paddle.fluid

### PR DESCRIPTION
**Pull request by ZHUI**
### PR types

Others
### PR changes

Others
### Description

Remove some code with paddle.fluid

### CODE REVIEW

File: examples/language_model/moe/dygraph/run_moe_pretrain.py [hunk 1]
```diff
@@ -527,6 +525,7 @@ def do_train(args):
         num_files = len(files)
         for f_id in range(file_id, num_files):
             data_file = files[f_id]
+            print(tokenizer.eos_token_id)
```
Comment (guoshengCS) [inline examples/language_model/moe/dygraph/run_moe_pretrain.py]: 这里是否是忘记删掉的呢

**Review by guoshengCS — DISMISSED** [review]

File: examples/language_model/moe/dygraph/run_moe_pretrain.py (hunk 1 above)
Comment (ZHUI) [reply examples/language_model/moe/dygraph/run_moe_pretrain.py]: done

**Review by guoshengCS — DISMISSED** [review]
LGTM

**Review by haohongxiang — APPROVED** [review]
LGTM for MoE

**Review by guoshengCS — APPROVED** [review]

**Pull request merged**